project/Build.scala

```diff
@@ -93,7 +93,7 @@ object Scrooge extends Build {
     libraryDependencies ++= Seq(
       "org.scalatest" %% "scalatest" % "2.2.4" % "test",
       "org.scalacheck" %% "scalacheck" % "1.12.2" % "test",
-      "junit" % "junit" % "4.10" % "test" exclude("org.mockito", "mockito-all")
+      "junit" % "junit" % "4.12" % "test"
```

Comment:
it's probably fine–we use 4.10 internally, but I don't think it'll cause any problems.  we probably can't use 4.12 features yet though.